kroxylicious-systemtests/src/test/java/io/kroxylicious/systemtests/extensions/TestKubeKmsFacadeInvocationContextProvider.java

```diff
@@ -61,7 +65,18 @@ public List<Extension> getAdditionalExtensions() {
                             return kmsFacade;
                         }
                     },
-                    (AfterEachCallback) extensionContext -> kmsFacade.stop());
+                    (AfterEachCallback) extensionContext -> {
+                        try {
+                            Optional<Throwable> exception = extensionContext.getExecutionException();
```

Comment:
It would be, but we need to assure that we get the dumps before the Kms is stopped (removed). If we move it to other `AfterEach` we cannot assure that one `AfterEach` is executed before that the others.

Reply:
Yes, you are right.